Sources/OpenAIKit/Chat/ChatWithImage.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  File.swift
+//  
+//
+//  Created by 贺峰煜 on 2023/11/29.
+//
+
+import Foundation
+
+public struct ChatWithImage {
+    public let id: String
+    public let object: String
+    public let created: Date
+    public let model: String
+    public let choices: [Choice]
+    public let usage: Usage
+}
+
+extension ChatWithImage: Codable {}
```

Comment:
This doesn't seem right, you should be able to extend the pre-existing Chat model to support the inclusion of an image.

Reply:
I revised the redundant parts of the code.